### ISSUE TITLE
Style for (sub)modules?

### ISSUE THREAD

**Issue by nickrobinson251** [invenia/BlueStyle#75]
e.g. don't indent inside `module ... end"?
```julia
# Yes
module Foo

export bar

bar() = 1

end

# No
module Foo

    export bar

    bar() = 1

end
```

Perhaps other rules? e.g. Blank line after `module` and before `end`?
```julia
module Foo

export bar
bar() = 1

end

# No
module Foo
export bar
bar() = 1
end
```

it'd also be kind nice to add a comment after `end` e.g.  `end  # module`?

Raised by @iamed2 in https://github.com/invenia/LibPQ.jl/pull/197#discussion_r494658414

**Comment by omus** [comment]
I agree with the suggestions made here. I'd also suggest one module definition per file.
